tests/specs/run/webtransport/main.ts

```diff
@@ -0,0 +1,100 @@
+// Copyright 2018-2024 the Deno authors. All rights reserved. MIT license.
+
+import { decodeBase64 } from "@std/encoding/base64";
+import { assertEquals } from "@std/assert";
+
+const cert = Deno.readTextFileSync("../../../testdata/tls/localhost.crt");
+const certHash = await crypto.subtle.digest(
+  "SHA-256",
+  decodeBase64(cert.split("\n").slice(1, -2).join("")),
+);
+
+const server = new Deno.QuicEndpoint({
```

Comment:
Could you wrap this whole thing in a `Deno.test()` so that we get the benefit of sanitizers?